README.rst

```diff
@@ -976,6 +980,15 @@ to new devices.
 This feature works also when editing group templates or the group assigned
 to a device via the `REST API <#change-device-group-detail>`__.
 
+Group Configuration Variables
+#############################
+
+Groups allow to define configuration variables which are automatically
```

Comment:
Let's mention variable precedence and refer to the configuration variables section for the full info.